### DISCUSSION TITLE
.net 6 contains out of date DigiCert intermediate certificates?

### DISCUSSION THREAD

**Discussion by mmmmunsta** [dotnet/dotnet-docker#4603] [Q&A]
Does anybody know how often the certs get updated? Will this be updated any time soon as we're seeing issues where the Certs are mismatching. Could imagine this being an issue for others also?

Many thanks

**Comment by mthalman** [comment]
Certificates are managed by the `ca-certificates` package which is provided by the various Linux distros. Which Linux distro (or more specifically, which image tag) is this in reference to?

**Reply by mmmmunsta** [reply]
Hi there, sorry i missed this from the post.

The image being used is:

mcr.microsoft.com/dotnet/aspnet:6.0


Many thanks!

**Reply by mthalman** [reply]
The image for that tag contains the latest available version of [`ca-certificates` for Debian 11 (bullseye)](https://packages.debian.org/bullseye/ca-certificates): version 20210119.
